### PR TITLE
AddIsLiftable and AddLift for FinSets

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

**Comment by mohamed-barakat** [comment]
I still need to add examples for the new code.

File: gap/FinSetsForCAP.gi [hunk 1]
```diff
@@ -685,6 +685,30 @@ AddIsSplitMonomorphism( FinSets,
     return IsInitial( Range( phi ) ) or ( not IsInitial( Source( phi ) ) and IsMonomorphism( phi ) );
 end );
 
+##
+AddIsLiftable( FinSets,
+  function ( beta, alpha )
```
Comment (zickgraf) [inline gap/FinSetsForCAP.gi:690]: Could you swap alpha and beta (here and also below)? The CAP documentation uses alpha as the first argument, and using a different order makes it hard to think about this code.

File: gap/FinSetsForCAP.gi [hunk 2]
```diff
@@ -685,6 +685,30 @@ AddIsSplitMonomorphism( FinSets,
...
+end );
+
+##
+AddLift( FinSets,
+  function ( beta, alpha )
+    local L, imgs, chi;
+    
+    L := AsList( Source( beta ) );
+    
+    imgs := List( L, beta ); ## subset of N = Range( beta ) = Range( alpha )
```
Comment (zickgraf) [inline gap/FinSetsForCAP.gi:703]: This is a very expensive way of more or less reconstructing `AsList( beta )`, or am I missing something?

**Review by zickgraf — CHANGES_REQUESTED** [review]

File: gap/FinSetsForCAP.gi (hunk 2 above)
Comment (mohamed-barakat) [reply gap/FinSetsForCAP.gi]: Good idea. See new push.

File: gap/FinSetsForCAP.gi (hunk 1 above)
Comment (mohamed-barakat) [reply gap/FinSetsForCAP.gi:690]: This is the convention of my lecture. The lift is the solution of the system χ α = β.

**Review by zickgraf — APPROVED** [review]

**Pull request merged**